main.c

```diff
@@ -135,6 +135,14 @@ main(int argc, char *argv[])
 	insecure_memzero(passwd, strlen(passwd));
 	free(passwd);
 
+	/* Close files if they are not streams. */
```

Comment:
All files are streams.  What you mean is "Close any files we opened".

I'd skip the braces for such trivial code btw, but I don't mind if you prefer to keep them in.